httpx/dispatch/connection_pool.py

```diff
@@ -70,6 +70,8 @@ def __len__(self) -> int:
 
 
 class ConnectionPool(Dispatcher):
+    KEEP_ALIVE_TIMEOUT = 5.0
```

Comment:
Should this be made configurable, e.g. as a new option on `Timeout()`?

Reply:
We might want to name it "expiry" rather than "timeout".
Yes ideally configurable, perhaps on the `PoolLimits`, but we could probably treat that as a second pass.